packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -95,6 +95,15 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    */
   scrollSelectedToTop?: boolean;
 
+  /**
+   * The index of the option to select (use selectedKey to match option's key)
+   */
+  selectedIndex?: number;
+
+  /**
+   * Set input field text
+   */
+  inputFieldText?: string;

```

Comment:
Unclear why you would want to control this. Don't we have an onRender* override if you really want to override the title rendering behavior?

Reply:
@dzearing Hi! I see the onRender props for the callout options (ISelectableDroppableTextProps), but I don't see anything that I could use for updating the input field. 
 I wanted to update the defaultVisibleValue in BaseAutoFill (in ComboBox), but it has a this._currentVisibleValue that changes the input text to match the text of the option that is selected (using _getVisibleValue() ).  If (in the screenshot above) the user types 2:35 PM and I choose to have the 2:30 PM option selected when they open the dropdown (the nearest approximate match to 2:35 PM), the current thing that happens is _getVisibleValue() overrides the user's input to 2:30 PM.